javascript/grid-ui/BUILD.bazel

```
@@ -78,3 +79,18 @@ zip_file(
         "//java/server/src/org/openqa/selenium/grid:__pkg__",
     ],
 )
+
+jest_test(
+    name = "test",
+    args = [
+        "--no-cache",
+        "--no-watchman",
+        "--ci",
+        "--colors",
+        "--config",
+        "jest.config.js"
```

```suggestion
        "$(location jest.config.js)"
```

That will find it local to the bazel build